### PR TITLE
fix: BOAS-11 namepath WIP

### PR DESCRIPTION
**Pull request by MariaValeroGonzalezKC**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [√ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [√ ] I have performed a self-review of my own code
- [√ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [√ ] I have provided details on how I have tested my code
- [√ ] I have referenced the ticket number above
- [√ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/api/src/server/app/state-machine/__tests__/full-planning-appeal.machine.test.js [hunk 1]
```diff
@@ -21,17 +21,23 @@ test.before('sets up mocking of actions', () => {
 });
 
 /**
- * @param {object} t unit test
- * @param {string} initialState initial state in state machine
- * @param {string} action action taken to proceed in state machine
- * @param {string} expectedState expected state after action was taken
- * @param {object} context Context of transition
- * @param {boolean} hasChanged True if action was valid, False if action was invalid
+ * @param {object} t
```
Comment (brpins) [inline apps/api/src/server/app/state-machine/__tests__/full-planning-appeal.machine.test.js:24]: This declaration is incorrect, but it's a jsdoc thing anyway which is an upcoming ticket we can address this in

Comment (MariaValeroGonzalezKC) [reply apps/api/src/server/app/state-machine/__tests__/full-planning-appeal.machine.test.js:24]: To be solved in BOAS-17

**Review by francescobrenci-AR — APPROVED** [review]

**Pull request merged**